### DISCUSSION TITLE
Artifact publishing to Maven Central

### DISCUSSION THREAD

**Discussion by mversaggi** [blackears/svgSalamander#74] [General]
Hi there,

Are there any plans to publish the latest 1.1 releases to Maven Central? I noticed that [1.0 is already published](https://mvnrepository.com/artifact/com.kitfox.svg/svg-salamander), but would like to be able to pull down the SVG Salamander 1.1 artifacts from Maven Central to avoid cluttering my project's repository with a local copy of the library (specifically 1.1.1, and possibly 1.1.2).

Thank you in advance!

**Comment by DevCharly** [comment]
I maintain a clone of this project and also publish it to Maven Central:
https://search.maven.org/artifact/com.formdev/svgSalamander

https://github.com/JFormDesigner/svgSalamander

**Reply by mversaggi** [reply]
Awesome, I'll go ahead and pull from your Maven Central publications for now. Thank you!

**Comment by blackears** [comment]
I tried to publish a new release a few months ago, but found the process really confusing and wasn't able to complete it.  I might try publishing on Github, but need to review that process too.  In the mean time, you may want to use a third party distribution.

**Reply by mversaggi** [reply]
Good to know, looks like I can pull from @DevCharly's publications in the com.formdev group for now, but I'll keep an eye out for future publications to Maven Central. Thank you!

**Comment by bradh** [comment]
@DevCharly Any chance you can update to 1.1.4?

**Reply by DevCharly** [reply]
Sure, it is now on Maven central: https://repo1.maven.org/maven2/com/formdev/svgSalamander/1.1.4/

**Comment by blackears** [comment]
Thanks.
